### DISCUSSION TITLE
Can't get loading from snippet files working with C header files

### DISCUSSION THREAD

**Discussion by ytkimirti** [L3MON4D3/LuaSnip#722] [Q&A] [answered]
Newbie here
I forked friendly snippets repo to modify some snippets and add my own. They work with `.c` files but not with `.h`.
I the package.json of friendly snippets I did
```json
      {
        "language": "c",
        "path": "./snippets/c.json"
      },
      {
        "language": "h",
        "path": "./snippets/c.json"
      },
```
but it doesn't work.

I do call `require("luasnip/loaders/from_vscode").load()` and all other snippets I add work. Just not with header files

**Comment by L3MON4D3** [comment, marked as answer]
Hi!
Are you sure the filetype is "h"? The `language`-entry does not necessarily match the file-extension, but the filetype (`echo &ft` to print it, I think)
If that wasn't it, try enabling logging (see other issues, there are some recent ones) and checking the logs, one common issue is malformed (due to a trailing comma🙄🙄) json

**Reply by ytkimirti** [reply]
Thank you so much. Actually the filetype of c header file was cpp when I echoed it. Makes sense now :D. I didn't know about filetypes in vim, have a great day :)
